### PR TITLE
Refactor the backend and improve building/linting in the app

### PR DESCRIPTION
**Pull request by typokign**
### Administrative Info
Monday Board ID: 853642866

### Changes
See commits.

### CODE REVIEW

**Review by Anshul-Birla — DISMISSED** [review]
looks good

**Review by allisonbhavsar — DISMISSED** [review]
lgtm!

**Review by Anshul-Birla — DISMISSED** [review]
lgtm!

File: backend/internal/database/db.go [hunk 1]
```diff
@@ -1,50 +1,32 @@
...
-	"log"
-	"os"
-	"strconv"
-
-	"github.com/jackc/pgx"
+	"github.com/TritonSE/words-alive/internal/utils"
 	"github.com/jackc/pgx/stdlib"
+	"github.com/jackc/pgx/v4/pgxpool"
 	"github.com/pressly/goose"
+	"log"
```
Comment (k6han) [inline backend/internal/database/db.go:10]: Should we organize the import statements here?

**Review by k6han — DISMISSED** [review]
LGTM

**Review by k6han — DISMISSED** [review]

**Review by Anshul-Birla — APPROVED** [review]
i think it looks good :)

**Review by aksharans — APPROVED** [review]
lgtm

**Pull request merged**